docs/developer/checkout/troubleshooting.mdx

```diff
@@ -11,3 +11,52 @@ sidebar_label: Troubleshooting
 ### Variant/Product is not available
 
 Make sure product is available in the channel the checkout is created. See [Product Troubleshooting](/developer/products/troubleshooting.mdx).
+
+### Resolving Paid Checkouts Not Converting to Orders
+
+Sometimes, a `Checkout` that has been paid might not convert into an `Order`.
+This can happen for various reasons, such as the customer closing the browser before the `checkoutComplete` mutation was called.
+To ensure that fully paid checkouts are reliably converted into orders, consider implementing one of the following solutions:
+
+#### Solution 1: Use a Webhook to Complete Checkouts
+
+Set up the `CHECKOUT_FULLY_PAID` webhook, and create an app to call the `checkoutComplete` mutation when this webhook is triggered.
+	•	For more information on setting up webhooks, refer to the [webhook overview](/developer/extending/webhooks/overview).
+	•	You can find guidance on app development [here](/developer/extending/apps/overview).
+
+	Note: This approach only works when the `CHARGE` is used as the default transaction flow strategy.
```

Comment:
I'm not sure we should recommend this approach, since queue might be overloaded - this has previously resulted in situations where webhooks weren't delivered for 6h+ and because of that our release funds task that refunds transactions for uncompleted checkouts.

This also contradicts our previous recommendations, for example [in migration guide](https://docs.saleor.io/upgrade-guides/core/3-19-to-3-20#automatic-checkout-completion):
> Previously a store might have had missing orders, due to checkoutComplete mutation not being always called. This could have happened for many unexpected reasons: network issues, storefront bug, customer leaving website, closing tab before being redirected from payment provider, etc.

Reply:
So maybe we should not delete this recommendation, but move it to the end and mention the limitations?